docs/section18/Rule_18-2.md

```diff
@@ -0,0 +1,80 @@
+# HVAC_SystemZoneAssignment - Rule 18-2 
+**Schema Version:** 0.0.28  
+**Mandatory Rule:** True  
+**Rule ID:** 18-2  
+**Rule Description:** Does the modeled system serve the appropriate zones (one system per zone for system types 1, 2, 3, 4, 9, 10, 11, 12, and 13 and one system per floor for system types 5, 6, 7, and 8, with the exception of system types 5 or 7 serving laboratory spaces - these systems should serve ALL laboratory zones in the buidling).  
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section:** Section 18 HVAC_SystemZoneAssignment  
+**Appendix G Section Reference:** Table G3.1.1.3  
+
+**Evaluation Context:** Each HVAC System  
+
+
+**Manual Check:** Yes  
+**Evaluation Context:** Evaluate whether each system serves the correct zones  
+**Data Lookup:**   
+**Function Call:** 
+
+1. get_baseline_system_types()
+2. get_zones_on_same_floor()
+3. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()
+4. baseline_system_type_compare()
+5. get_zone_target_baseline_system()
+6. get_building_total_lab_exhaust_from_zone_exhaust_fan()
+
```

Comment:
Supriya proposed at the last call to have a separate function to check if all lab zones are served by the same HVAC system. Since this is also used in Rule 18-23, perhaps we should do that. It will also simplify verification for this rule.